relatedq.do

```diff
@@ -0,0 +1,17 @@
+// related information might be useful
```

Comment:
Brilliant!! Let's keep them here. We definitely need to discuss this along with other surveys at a later stage. MICS are just not very standardised on HIV and related info.